packages/combobox/src/Combobox.tsx

```diff
@@ -759,9 +759,15 @@ export function Combobox<M extends boolean>({
         updateSelection(null);
         onClear?.(e);
         onFilter?.('');
+        setInputFocus();
       }
     };
 
+    const handleClearButtonKeyDown = (e: React.KeyboardEvent) => {
+      e.stopPropagation();
+      setInputFocus();
+    };
+
```

Comment:
I think this is overriding the `click` behavior when using the keyboard. I think anything other than `stopPropagation` should be handled in the click handler.

(Actually, if we want the arrow key navigation, I'm not even sure we want to stop propagation)


https://user-images.githubusercontent.com/2414030/218544399-a8cca362-986b-4ae3-ad2b-310aca7d4577.mov

Reply:
stopping prop doesn't prevent the keyboard nav

Reply:
It should though? Since the Arrow Key nav is handled via a `onKeyDown` handler on the wrapper, if we stop prop. on the button, the wrapper won't receive the event

Reply:
Those are separate events though. We don't want onClear opening the menu